internal/server/data/query.go

```diff
@@ -79,22 +57,31 @@ func insert(tx WriteTxn, item Insertable) error {
 	}
 	setOrg(tx, item)
 
-	query := Query("INSERT INTO")
+	query := querybuilder.New("INSERT INTO")
 	query.B(item.Table())
 	query.B("(")
-	query.B(columnsForInsert(item.Columns()))
+	query.B(columnsForInsert(item))
 	query.B(") VALUES (")
-	query.B(placeholderForColumns(item.Columns()), item.Values()...)
+	query.B(placeholderForColumns(item), item.Values()...)
 	query.B(");")
 	_, err := tx.Exec(query.String(), query.Args...)
 	return err
 }
 
-func columnsForInsert(columns []string) string {
-	return strings.Join(columns, ", ")
+// columnsForInsert is a privileged function that is not checked by
+// internal/tools/querylinter. If the arguments to this function change
+// the linter will likely need to be updated.
+// The return value must only include trusted strings.
```

Comment:
What is a trusted string in this context?

Reply:
Good question, it should mean "not from a user", as user input is never trusted. I'll think about how to make this more clear.

Reply:
I've updated this to:

> // The return value must only include trusted strings from the source code,
> // never untrusted user input.